src/containers/ClassifierContainer.jsx

```diff
@@ -168,11 +169,24 @@ class ClassifierContainer extends React.Component {
       </main>
     );
   }
+  
+  //----------------------------------------------------------------
+  
+  componentWillReceiveProps(next) {
+    if (!this.props.selectedAnnotation && next.selectedAnnotation) {
+      this.setState({
+        popup: <SelectedAnnotation annotation={next.selectedAnnotation} />
+      });
+    }
+  }
 
   //----------------------------------------------------------------
 
   closePopup() {
     this.setState({ popup: null });
```

Comment:
I'm working on a current PR that also moves popup state into redux, so these might change a bit in the future (after this merges). However, it's worth deciding here if we should keep popup state in each individual component.